Userland/Libraries/LibWeb/SVG/SVGGraphicsElement.cpp

```diff
@@ -116,42 +116,42 @@ Gfx::AffineTransform SVGGraphicsElement::get_transform() const
-            if (auto fill_value = parse_css_value(parsing_context, value, CSS::PropertyID::Fill))
-                style.set_property(CSS::PropertyID::Fill, fill_value.release_nonnull());
-        } else if (name.equals_ignoring_ascii_case("stroke"sv)) {
-            // FIXME: The `stroke` attribute and CSS `stroke` property are not the same! But our support is limited enough that they are equivalent for now.
-            if (auto stroke_value = parse_css_value(parsing_context, value, CSS::PropertyID::Stroke))
-                style.set_property(CSS::PropertyID::Stroke, stroke_value.release_nonnull());
-        } else if (name.equals_ignoring_ascii_case("stroke-width"sv)) {
-            if (auto stroke_width_value = parse_css_value(parsing_context, value, CSS::PropertyID::StrokeWidth))
-                style.set_property(CSS::PropertyID::StrokeWidth, stroke_width_value.release_nonnull());
-        } else if (name.equals_ignoring_ascii_case("fill-rule"sv)) {
-            if (auto fill_rule_value = parse_css_value(parsing_context, value, CSS::PropertyID::FillRule))
-                style.set_property(CSS::PropertyID::FillRule, fill_rule_value.release_nonnull());
-        } else if (name.equals_ignoring_ascii_case("fill-opacity"sv)) {
-            if (auto fill_opacity_value = parse_css_value(parsing_context, value, CSS::PropertyID::FillOpacity))
-                style.set_property(CSS::PropertyID::FillOpacity, fill_opacity_value.release_nonnull());
-        } else if (name.equals_ignoring_ascii_case("stroke-opacity"sv)) {
-            if (auto stroke_opacity_value = parse_css_value(parsing_context, value, CSS::PropertyID::StrokeOpacity))
-                style.set_property(CSS::PropertyID::StrokeOpacity, stroke_opacity_value.release_nonnull());
-        } else if (name.equals_ignoring_ascii_case(SVG::AttributeNames::opacity)) {
-            if (auto opacity_value = parse_css_value(parsing_context, value, CSS::PropertyID::Opacity))
-                style.set_property(CSS::PropertyID::Opacity, opacity_value.release_nonnull());
-        } else if (name.equals_ignoring_ascii_case("text-anchor"sv)) {
-            if (auto text_anchor_value = parse_css_value(parsing_context, value, CSS::PropertyID::TextAnchor))
-                style.set_property(CSS::PropertyID::TextAnchor, text_anchor_value.release_nonnull());
-        } else if (name.equals_ignoring_ascii_case("font-size"sv)) {
-            if (auto font_size_value = parse_css_value(parsing_context, value, CSS::PropertyID::FontSize))
-                style.set_property(CSS::PropertyID::FontSize, font_size_value.release_nonnull());
-        } else if (name.equals_ignoring_ascii_case("mask"sv)) {
-            if (auto mask_value = parse_css_value(parsing_context, value, CSS::PropertyID::Mask))
-                style.set_property(CSS::PropertyID::Mask, mask_value.release_nonnull());
+        for (auto property : attribute_style_properties) {
```

Comment:
We can iterate with `auto const& property` here to avoid copy-constructing the `FlyString` on each iteration :)

Reply:
I've removed the FlyString here as I realized it's really helping, as doing a `equals_ignoring_ascii_case()` prevents the pointer comparison.

Reply:
Good point!